easy/solution_12.sql

```diff
@@ -0,0 +1,9 @@
+-- 
+-- FInd tallest athlete
+SELECT 
+    name, 
+    height 
+FROM public.olympics_history
+WHERE height IS NOT NULL AND height != 'NA'
+ORDER BY height DESC
+LIMIT 1;
```

Comment:
Same issue as that of youngest athlete. LIMIT 1 selects only one while there could be more than 1 athlete with the tallest height.